main.go

```diff
@@ -120,13 +122,16 @@ func main() {
 		// the gRPC server must be started without credentials.
 		grpcServer = web.GRPCServer(unaryOptions, streamOptions)
 	}
-
 	qfService := web.NewQuickFeedService(logger, db, scmManager, bh, runner)
 	if err := qfService.InitSCMs(context.Background()); err != nil {
 		log.Fatalf("Failed to initialize SCM clients: %v", err)
 	}
 
 	qf.RegisterQuickFeedServiceServer(grpcServer, qfService)
+	if err = web.VerifyAccessControlMethods(grpcServer); err != nil {
```

Comment:
Minor: would it be possible to call it earlier or must we first call `RegisterQuickFeedServiceServer`?

Reply:
I think we need to register first, but I'll check.

Reply:
Just like I expected, getting `gRPC server missing qf.QuickFeedService service` without registering (it is after all just an empty gRPC server at this point).